### PR TITLE
Created a general Meetings page & a 2020 Bryn Mawr Meetings page

### PR DESCRIPTION
**Pull request by rocco8773**
What was done...

1. Create a general meetings page for upcoming and past meetings.
1. Added the meetings page to the top menu `Community/Meetings`
1. Added a meeting page specifically for the 2020 Bryn Mawr Meeting. I'm currently calling the meeting the "**1st PlasmaPy Mini-Conference**."
1. Updated `custum.css` with formatting for tables. Before the boarders were invisible.
1. Removed the search bar, since it's still not aesthetic.  I think we have too many menus in the top-menu so the search button to the search field is being moved underneath the search field.

To navigate to the "**1st PlasmaPy Mini-Conference**" page, you first have to navigate to the general meetings page.

I'd like to get a highlight/featured section on the PlasmaPy landing page, but we should leave this for a separate PR.

### CODE REVIEW

**Comment by StanczakDominik** [comment]
I'm sorry for this search form back-and-forth, but I found out about https://plugins.getnikola.com/v7/localsearch/ and wanted to try my hand at implementing it. I think you'll find it solves out problems. Seems like the power of javascript knows no bounds!

File: web/conf.py [hunk 1]
```diff
@@ -23,10 +23,10 @@
 # -- Links --
 # This is the main URL for your site. It will be used
 # in a prominent link. Don't forget the protocol (http/https)!
-SITE_URL = "https://plasmapy.github.io/"
+SITE_URL = "https://plasmapy.org/"
```
Comment (StanczakDominik) [inline web/conf.py]: I had to change this up so the URLs in search would come out right.

... I hope it didn't break anything.

File: web/conf.py [hunk 2]
```diff
@@ -1214,6 +1214,11 @@
 # it appears on the navigation bar:
 #
 # SEARCH_FORM = """
+SEARCH_FORM = """
+<form class="navbar-search pull-right" action="/search/" role="search">
+    <input type="text" class="form-control" id="tipue_search_input" name="q" placeholder="Search" autocomplete="off">
+</form>
+"""
```
Comment (StanczakDominik) [inline web/conf.py]: I mostly used https://github.com/mate-desktop/mate-desktop.org/blob/8552877b8cdfc8ad28b9e4c4b67f0de10fc7e4dc/conf.py#L1355-L1363 to throw this together.

**Comment by rocco8773** [comment]
Nice! This definitely looks cleaner in the top-menu.  I also like how the search stays within the PlasmaPy setting.

I'm coming across two issues though.

1. The search does not seem to be very exhaustive. If I do a test search for "about", I don't get the about page as a search result.
1. Whenever I click on one of the links after a search, I get a "Warning: Potential Security Risk Ahead" from Firefox.  I'm running this from a local build, so I don't know if the issue will persist on a deployed build.  I'm guessing this problem is arising from an certificate issue, just my guess.

**Comment by rocco8773** [comment]
@StanczakDominik Since we want to get this PR merged before sending out invites to our upcoming workshop, I'm going to split off the search field into a separate PR.

File: web/pages/meetings/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+author: Erik T. Everson
+
+### Upcoming Meetings
+
+----
+
+#### [2020 April:  1st PlasmaPy Workshop](1st_workshop_2020_at_bryn_mawr)
+April 20-22, 2020
+
+PlasmaPy's first mini-conference.  The conference is setup as
```
Comment (namurphy) [inline web/pages/meetings/index.md]:
```suggestion
PlasmaPy's first collaboration meeting.  The conference is set up as
```

File: web/pages/meetings/index.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+a 3 day meeting for PlasmaPy developers and the PlasmaPy community. The first day (4/20) is 
+designated for the PlasmaPy Development Team and the remaining two days (4/21-4/22) are 
+designed for community interaction.
```
Comment (namurphy) [inline web/pages/meetings/index.md]:
```suggestion
designated for the PlasmaPy Team to discuss the nuts and bolts of code development,
and the remaining two days (4/21-4/22) are designed for community interaction.
```

File: web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
+title:
```
Comment (namurphy) [inline web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]:
```suggestion
title: 1st PlasmaPy Workshop
```

File: web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+[Agenda](#Agenda)<br/>
+[Registration]()<br/>
+[Travel & Housing](#Travel-Housing)<br/>
+[Travel Funding for Students](#Travel-Funding-for-Students)<br/>
+[Web Participation](#Web-Participation)<br/>
+[Support & Acknowledgements](#Support-Ack)
+
+### <a name="Overview"></a> Overview
+
+PlasmaPy's first mini-conference.  The conference is setup as a 3 day meeting for
```
Comment (namurphy) [inline web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]:
```suggestion
PlasmaPy's first collaboration meeting.  The conference is set up as a 3 day meeting for 
```

File: web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+### <a name="Overview"></a> Overview
+
+PlasmaPy's first mini-conference.  The conference is setup as a 3 day meeting for 
+PlasmaPy developers and the PlasmaPy community. The first day (4/20) is 
+designated for the PlasmaPy Development Team and the remaining two days (4/21-4/22) are 
+designed for community interaction.  Goals of the meeting are to (1) further develop
+PlasmaPy's roadmap, (2) inform the broader PlasmaPy community of PlasmaPy's objectives 
+and direction, (3) demonstrate PlasmaPy's capabilities to the community, and 
+(4) receive valuable feedback from the community.<br/><br/>
```
Comment (namurphy) [inline web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]:
```suggestion
designated for the PlasmaPy Team to discuss the nuts and bolts of code development
while the remaining two days (4/21-4/22) are designed for community interaction.  
The goals of the meeting are to (1) further develop PlasmaPy's roadmap, 
(2) inform the broader community of PlasmaPy's objectives and direction, 
(3) demonstrate PlasmaPy's capabilities, and (4) receive valuable feedback from 
the community.<br/><br/>
```

File: web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+### <a name="Agenda"></a> Agenda
+
+---
+**NOTE**
+
+As the meeting approaches, please check the agenda for updates.  As presentation material
+becomes available we will post links for slides and such.
```
Comment (namurphy) [inline web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]:
```suggestion
becomes available we will post links for slides and additional information.
```

File: web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+Day 1 is intended for the PlasmaPy committee and internal topics.  Attendees will be
+restricted to the PlasmaPy committee and developers.  We'll address topics related to 
+package development, community outreach, and education.
+
+| Time | Section | Title | Speaker/Leader |
+| --- | --- | --- | --- |
+| 08:30 - 09:00 | 0  | **ARRIVAL & COFFEE** |  |
+| 09:00 - 09:15 | 1  | Welcome: Introduction & Overview |  |
+| 09:15 - 09:35 | 2  | Roadmap: Overview & Current Projections |  |
+| 09:35 - 10:35 | 3  | Roadmap: Formulary Development |  |
```
Comment (namurphy) [inline web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]: I don't think we'll need as much time for formulary development as for simulations and experiments.

File: web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md (hunk 5 above)
Comment (rocco8773) [reply web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]: I purposely left this meta field blank.  I don't like the heading it creates and the heading provides no functionality anyways.

I have the page heading as the top-level markdown heading.

File: web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md (hunk 9 above)
Comment (rocco8773) [reply web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]: Oh yeah, I originally had 1 hr for every roadmap session and, I guess, this one was missed when I changed to 45 min sessions.

Comment (rocco8773) [reply web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]: Looks like we get a slightly longer lunch. I don't have a problem with that!😁

**Comment by StanczakDominik** [comment]
@rocco8773 oh yeah, sure, I should have done that in the first place. Sorry!

File: web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+<br/>
+
+### <a name="Travel-Housing"></a> Travel & Housing
+
+---
+**NOTE**
+
+Due to the ongoing development of the novel coronavirus (COVID-19), travel restrictions
+may be in place when this workshop occurs.  If this is the case, we will still proceed 
+with the workshop via a [web-cast](#Web-Participation).
```
Comment (namurphy) [inline web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md]: Here we should recommend strongly against purchasing nonrefundable plane tickets.

**Pull request merged**